PVList.md

```diff
@@ -0,0 +1,301 @@
+DIS=CO
+AREA={IA}-{{01..20}RaBPM|20RaBPMTL}
+# MCH
+## $(AREA):$(DIS)-MCHBase:
```

Comment:
Use a code block for the header, otherwise github's renderer interprets it as latex math

I'm not a huge fan of so much HTML in an MD file, but it *is* useful.

Reply:
> I'm not a huge fan of so much HTML in an MD file, but it is useful.

In this case, is it ok?